### ISSUE TITLE
``filter-++`` ?

### ISSUE THREAD

**Issue by mechvel** [agda/agda-stdlib#934]
I do not find this one in lib-1.1:
```
filter-++ :  ∀ (p? : Decidable P) xs ys → 
                 filter p? (xs ++ ys) ≡  filter p? xs ++ filter p? ys
```
Probably, it is needed.

**Comment by MatthewDaggitt** [comment]
Sure, feel free to open a PR.

**Comment by MatthewDaggitt** [comment]
Closed in favour of #933
